.eslintrc.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
         '@typescript-eslint/explicit-module-boundary-types': 'off',
         '@typescript-eslint/no-empty-function': 'off',
         'import/extensions': 'off',
+        'import/prefer-default-export': 'off',
```

Comment:
This is widely used as we define constants in a separate file and use default export the defined constants as an object. 
May I know why this is turned off? Any blockers?

Reply:
This rule is turned off exclusively for typescript files.

As it is declared, it forces any js/ts file to declare a default export. But it is not always useful and most of the time one or multiple named export is better to stay consistent across the project. So I don't see why we force a default export in every file.

Reply:
I think we can have uniformity and prefer to have default export from all the files. @chipp972

Reply:
Maybe uniformity is good when we don't have types and rely on naming convention and uniform file structure. But in Typescript, it is not very relevant and encourages bad behaviors like renaming a component across the project.

But ok I put back the rule